lib/AppInfo/Application.php

```diff
@@ -56,6 +56,9 @@ public function register(IRegistrationContext $context): void {
 		}
 
 		$context->registerEventListener(UserDeletedEvent::class, UserDeletedListener::class);
+
+		$manager = \OC::$server->get(\OCP\Notification\IManager::class);
+		$manager->registerNotifierService(\OCA\Calendar\Notification\Notifier::class);
```

Comment:
**nitpick:** You can use the `registerNotifierService` directly from the `$context` parameter.